### DISCUSSION TITLE
How to switch to different node version?

### DISCUSSION THREAD

**Discussion by ximex** [actions/virtual-environments#9057] [Installed software]
get following error message when i try to link `node@20`: `Error: No such keg: /usr/local/Cellar/node@20`

i tried:
```sh
brew unlink node // tried with and without
brew link node@20 --force --overwrite // tried also with full version 20.10.0
```

running `node --version` gives: `v18.19.0`

running `brew --prefix` gives: `/usr/local`

running `brew search node` gives:
```
aws/tap/eks-node-viewer
libbitcoin-node
linode-cli
llnode
node
node-build
node-sass
node@14
node@16
node@18
node@20
node_exporter
nodebrew
nodeenv
nodenv
ode
```

**Comment by shamil-mubarakshin** [comment]
Hello @ximex,
If using GitHub Actions, could you try adding [setup-node](https://github.com/actions/setup-node) to your workflow:
```
- uses: actions/setup-node@v4
  with:
    node-version: 20
```

If using AzDO Pipelines, try adding [UseNode](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-node-v1?view=azure-pipelines) to your workflow:
```
- task: UseNode@1
  inputs:
    version: '20.x'
```

**Comment by ximex** [comment]
forgot to say that we use [appcenter](https://appcenter.ms/). so is it possible with the post-clone (`appcenter-post-clone.sh`) script? https://learn.microsoft.com/en-us/appcenter/build/custom/scripts/

**Reply by shamil-mubarakshin** [reply]
In this case, depending on your project, something like below should install and link node20:
```
brew install node@20
brew link node@20 --force --overwrite
```
